### DISCUSSION TITLE
DAHENG IMAGING

### DISCUSSION THREAD

**Discussion by dentx22** [jakobwilm/slstudio#52] [General]
I'm trying to add these cameras but no matter how hard I try I can't.
can you insert them?
this is the link of the sdk thanks
these cameras have them many people are good for both david 4 and 5
and also for the ScanCenter NG
currently the most used programs for do-it-yourself scans

https://www.get-cameras.com/customerdownloads

thanks

**Comment by jakobwilm** [comment]
Hi.
Currently I could only do this on a contract basis. Rough estimate: 2000Euro. I would also need access to one of those cameras.
Let me know if this is of interest to you.
Sincerely
